app/application.py

```diff
@@ -52,6 +51,24 @@ def hello_world():
     return render_template('_temp.html', _theme='default')
 
 
+def build_survey_data(statusFilter):
+
+    # TODO - Derive the Party Id
```

Comment:
A very picky point. A link to the confluence page would be nice to let the user know where the current details are for this.